### ISSUE TITLE
Error installing the script subselect.lua

### ISSUE THREAD

**Issue by RaXorX** [zenyd/mpv-scripts#15]
I tried to install this script but it ends up giving me an error for tkinter "No module named tkinter."
I've already installed tk and tk-tools along with python-mpv (for the next error)

Additionally after I click okay on the error I get another error as python not found. This is also the case in mpv/mpvnet

I also configured the subselect.conf file appropriately but alas, same error as before. Although, the subselect.py runs normally and shows me a an input box for entering the name of the subtitles.

![image](https://user-images.githubusercontent.com/44987810/69484899-863bc480-0e5e-11ea-82d6-9ee0e7cdabb7.png)

Edit: Script works fine, except for inside mpv/mpvnet.
![image](https://user-images.githubusercontent.com/44987810/69484946-f0546980-0e5e-11ea-91d4-b81c5c2719ba.png)

**Comment by zenyd** [comment]
I didn't try with mpv.net but mpv opens the window fine. Maybe something with your installation is broken. Can you try de- and reinstalling python and with mpv instead of mpv.net?

You only need [https://www.python.org/ftp/python/3.8.2/python-3.8.2.exe](https://www.python.org/ftp/python/3.8.2/python-3.8.2.exe) (make sure to set the checkbox for putting python in the PATH) and subliminal. The python installation already includes tkinter.

**Comment by RaXorX** [comment]
Sorry, I never replied and forgot about it. I got it to work that time, apparently. There were two things necessary.
Had to change the directory where it would look for the files.

`if package.config:sub(1,1) == "/" then
   options.subselect_path = utils.join_path(os.getenv("HOME"), ".config/mpv.net/scripts/subselect.py")
   ops = "unix"
else
   options.subselect_path = utils.join_path(os.getenv("APPDATA"), "mpv.net\\scripts\\subselect.py")
   ops = "win"
end`

and then I was getting an error for python version not being detected. 
`if python_version.stdout:find("3%.") ~= nil then
         python = "python"`
At this point, it wouldn't detect my python version and give me an error corresponding to it.
So, I changed this line to work even if Python wasn't being found.
` if python ~= nil then
      ret = utils.subprocess({ args = { "python", options.subselect_path, video, options.down_dir, options.sub_language }})`

I changed that ~ to = so, if the version wasn't getting detected, it would still work. And it was working fine for me at that time.
